### PR TITLE
Added SDN Dashboard helm chart

### PR DESCRIPTION
**Pull request by csahithi**
#### What this PR does and why is it needed

This PR adds the helm chart for SDN dashboard corresponding to various components of ovn-kubernetes

#### Details to documentation updates

Documentation on how to install the dashboard has been added

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Comment by csahithi** [comment]
@girishmg pls review this PR

**Comment by csahithi** [comment]
ovn-host-controller-dashboard
cluster-manager-dashboard
north-daemon-dashboard

**Comment by tssurya** [comment]
can we please add docs to https://ovn-kubernetes.io/observability/grafana-and-prometheus/ for this? @girishmg this is equally important for us as the feature ;) you know why.. :D

File: helm/sdn-dashboard/README.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+# sdn-dashboard
+
+-----------------------
+
+![Version: 1.0.0](https://img.shields.io/badge/Version-1.0.0-informational?style=flat-square) ![AppVersion: 1.0.0](https://img.shields.io/badge/AppVersion-1.0.0-informational?style=flat-square)
```
Comment (tssurya) [inline helm/sdn-dashboard/README.md:5]: add a hook to website

File: helm/sdn-dashboard/README.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+- [grafana/grafana](https://github.com/grafana/helm-charts/tree/main/charts/grafana)
+
+After configuring Prometheus and Grafana and installing sdn-dashboard helm chart, ovn-kubernetes can be installed and the metrics will be scraped.
+
+## Port Forwarding to access Grafana and Prometheus UI
+
+```
+kubectl -n <desired_namespace> port-forward deployment/kube-prometheus-stack-grafana 3000:3000 --address 0.0.0.0
+
+kubectl -n <desired_namespace> port-forward prometheus-kube-prometheus-stack-prometheus-0 9090:9090 --address 0.0.0.0
```
Comment (flavio-fernandes) [inline helm/sdn-dashboard/README.md:86]: NIT: for the sake of making it easier for folks trying it out for the first time, would it be useful to mention the default username and password ( admin / prom-operator ) ?

**Comment by flavio-fernandes** [comment]
Depends on: https://github.com/ovn-org/ovn-kubernetes/pull/4411
Tested it on my helm deployment. Short of requests on doc, /lgtm !

File: helm/sdn-dashboard/README.md (hunk 2 above)
Comment (csahithi) [reply helm/sdn-dashboard/README.md]: Added username and password to the documentation

File: helm/sdn-dashboard/README.md (hunk 1 above)
Comment (csahithi) [reply helm/sdn-dashboard/README.md]: Added moved docs to docs/installation/helm/sdn-dashboard.md and added hook to mkdocs.yml

**Comment by girishmg** [comment]
> can we please add docs to https://ovn-kubernetes.io/observability/grafana-and-prometheus/ for this? @girishmg this is equally important for us as is the feature ;) you know why.. :D

Absolutely, @csahithi  will be working on it next.

**Review by girishmg — APPROVED** [review]
Looks like Flavio also has tested this.

**Comment by girishmg** [comment]
The failure in CI is not related to this PR since this PR just adds an Helm Chart for SDN dashboards, so going to bring this in.

**Pull request merged**